src/main/java/uk/gov/hmcts/probate/services/submit/clients/CoreCaseDataMapper.java

```diff
@@ -503,5 +533,46 @@ private String formatDate(String originalDateStr){
         }
         return "";
     }
+
+    public Optional<JsonNode> documentUploadMapper(JsonNode probateData, String fieldname) {
+        Optional<JsonNode> ret = Optional.empty();
+        Optional<JsonNode> documentUploads = Optional.ofNullable(probateData.get(fieldname));
+        if (documentUploads.isPresent()) {
+            ArrayNode documentUploadCcdFormat = new ObjectMapper().createArrayNode();
```

Comment:
ObjectMapper is a threadsafe singleton and should not be instantiated for each serilalization request. This should be spring wired and reused accordingly ie. 
@Autowired
ObjectMapper mapper;